page-campaigns.php

```diff
@@ -112,6 +116,7 @@
                 </p>
             </div>
         </div>
+		<?php if ($incoming_campaign || $current_campaign): ?>
```

Comment:
We should not be leaving tags open / closed on conditionals.  It's extremely hard to read and leaves the template left open to broken markup.  This has been an issue on the a few of the event templates.

Reply:
This has been re-factored! Thanks!